docs/guides/azure-authenticate-with-oidc.md

```diff
@@ -0,0 +1,199 @@
+
+### Configure the Databricks provider to use the service principal
+
+In your Terraform configuration, configure the Databricks provider to use the service principal.
+
+```hcl
+provider "databricks" {
+  azure_client_id = "<application-id>"
+  azure_tenant_id = "<tenant-id>"
+  host = "https://<workspace-url>"
+}
+```
+
+### Create a GitHub Action that authenticates to Azure Databricks
+
+To create a GitHub Action, make a `.github/workflows/deploy.yml` file in your repository.
+
+To authenticate to Azure Databricks using OIDC, ensure that your action has the `id-token: write` permission. You can then authenticate to Azure using the `azure/login` action. Finally, run `terraform apply` with the `azure/cli` action.
+
+```yaml
+name: Deploy to Azure Databricks
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    environments: production
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Terraform
+        uses: hashicorp/setup-terraform@v3
+
+      - name: Apply Terraform
+        run: |
+          terraform init
+          terraform plan
+          terraform apply -auto-approve
+        working-directory: path/to/terraform/module
+```
+
+### (Optional) GitHub Actions Details
+
+The `subject` field is used to scope the federated credentials to a specific GitHub Actions environment. The `subject` field is a string in the format `repo:<organization>/<repo>:environment:<environment>`. The `organization`, `repo`, and `environment` fields should be replaced with the appropriate values.
+
+If the action runs without an environment context, the `subject` field should be set to `repo:<organization>/<repo>:ref:refs/heads/<BRANCH-NAME>` if the workflow is triggered from a branch, or `repo:<organization>/<repo>:ref:refs/tags/<TAG-NAME>` when triggered from a tag.
+
+If needed, it is also possible to configure the `subject` field for your organization or repository. See the [GitHub Actions OIDC documentation](https://docs.github.com/en/actions/security-for-github-actions/security-hardening-your-deployments/about-security-hardening-with-openid-connect) for more information about how to configure the `subject` field.
+
+## Azure DevOps
+
+In Azure DevOps, you can use Workload Identity federation to authenticate to Azure Databricks using OIDC. This allows you to authenticate to Azure Databricks using federated credentials issued by Azure DevOps. Today, the Terraform Provider for Databricks leverages the Azure CLI to use workflow identity federation in Azure DevOps. This guide will walk you through the steps to authenticate to Azure Databricks using OIDC on Azure DevOps.
+
+### Configure a service connection for your DevOps pipeline
+
+First, you need to create a service connection with federated credentials. This service principal will be used to authenticate to Azure Databricks. You can create a service principal using the `azuread` Terraform provider.
```

Comment:
nit: You can create `it` instead of `service principal` (as we already say service principal before).